include/OversamplingHelpers.h

```diff
@@ -0,0 +1,226 @@
+/*
+ * OversamplingHelpers.h
+ *
+ * Copyright (c) 2023 Lost Robot <r94231/at/gmail/dot/com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef OVERSAMPLING_HELPERS_H
+#define OVERSAMPLING_HELPERS_H
+
+#include <algorithm>
+#include <array>
+
+#include "hiir/PolyphaseIir2Designer.h"
+#include "hiir/Downsampler2xFpu.h"
+#include "hiir/Upsampler2xFpu.h"
+
+constexpr float HIIR_DEFAULT_PASSBAND = 19600;
+constexpr int HIIR_DEFAULT_MAX_COEFS = 8;
```

Comment:
```suggestion
inline constexpr float HIIR_DEFAULT_PASSBAND = 19600;
inline constexpr int HIIR_DEFAULT_MAX_COEFS = 8;
```
Namespace-scope constants in headers should be `inline` to avoid potential ODR issues.
https://stackoverflow.com/a/60963796/8704745